frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -62,7 +62,7 @@
 
 <div id="header"></div>
 <p class="pageHeader">
-    <span class="pageHeaderBigSpan">Components</span> <span class="pageHeaderSmallSpan">(${componentList.size()})</span>
+    <span class="pageHeaderBigSpan">Components</span> <span class="pageHeaderSmallSpan" id="componentCounter">(${componentList.size()})</span>
```

Comment:
I believe it would be nicer to have another `<span>` with just the number, without parantheses. Then you wouldn't have to strip the parentheses and add them back. The current way duplicates the rule for displaying the component count. It will create a conflict with #476 which will be hard to reconcile. If you introduce a span for just the number, merging with #476 will be a breeze.